app/controllers/pages_controller.rb

```diff
@@ -1,7 +1,5 @@
 class PagesController < SlicesController
 
-  caches_page :virtual_error_pages
```

Comment:
(Maybe something to add to the upgrade guide, if the idea is that the app embedding slices has to do it itself now.)